Apigee/Mint/DataStructures/DataStructure.php

```diff
@@ -107,4 +108,34 @@ public function __get($name)
             return null;
         }
     }
+
+    public function __toString()
+    {
+        $data = array();
+        $reflect = new \ReflectionClass($this);
```

Comment:
Reflection is the proper way to access properties of a subclass in the parent class.